includes/components/class-attribute.php

```diff
@@ -800,7 +800,7 @@ public function add_admin_fields( $meta_box ) {
 		$model = $meta_box['model'];
 
 		// Get category IDs.
-		$category_ids = $this->get_category_ids( $model, get_the_ID() );
+		$category_ids = hp\get_array_value( $meta_box, 'category', $this->get_category_ids( $model, get_the_ID() ) );
```

Comment:
Also check the comment below about the request context, maybe we can avoid adding another argument to the meta box function in Admin and pass the category ID another way